cmd/oras/copy.go

```diff
@@ -132,6 +145,7 @@ func runCopy(opts copyOptions) error {
 			copyOptions := oras.CopyOptions{
 				CopyGraphOptions: extendedCopyOptions.CopyGraphOptions,
 			}
+			copyOptions.Concurrency = opts.concurrency
```

Comment:
This line is redundant as `copyOptions` is derived from `extendedCopyOptions`.

Reply:
done